visualization/app/codeCharta/codeCharta.model.ts

```diff
@@ -250,6 +258,7 @@ export interface GlobalSettings {
 	experimentalFeaturesEnabled: boolean
 	layoutAlgorithm: LayoutAlgorithm
 	maxTreeMapFiles: number
+	sharpnessMode: SharpnessMode
```

Comment:
We should check that the new option is stored through different sessions: see #1587